backend/FwLite/FwDataMiniLcmBridge/LcmUtils/ProjectLoader.cs

```diff
@@ -16,8 +17,12 @@ public interface IProjectLoader
 
 public class ProjectLoader : IProjectLoader
 {
-    public const string ProjectFolder = @"C:\ProgramData\SIL\FieldWorks\Projects";
-    private static string TemplatesFolder { get; } = @"C:\ProgramData\SIL\FieldWorks\Templates";
+    public static string DataFolder =
+        RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ?
+            @"C:\ProgramData\SIL\FieldWorks" :
+            Path.Join(Environment.GetEnvironmentVariable("XDG_DATA_HOME") ?? Path.Join(Environment.GetEnvironmentVariable("HOME"), ".local", "share"), "fieldworks");
```

Comment:
This is a bit unwieldy. I could pull this out into another property called `LinuxDataFolder` that gets the XDG_DATA_HOME env var and falls back to the HOME env var if XDG_DATA_HOME is not defined. (Which usually means OS X, as the XDG standard is supported across all major Linux distros, but Apple doesn't implement it).

Reply:
yeah break it out into `UnixDataFolder`. We can split out a mac specific one if needed later.

Reply:
Done in commit d66df8a2.